suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -273,22 +273,36 @@ export const acquireDevice = createThunk(
  * Called from `discoveryMiddleware`
  * Fetch device state, update `devices` reducer as result of SUITE.AUTH_DEVICE
  */
-export const authorizeDevice = createThunk(
+type AuthorizeDeviceParams = { shouldIgnoreDeviceState: boolean } | undefined;
+export type AuthorizeDeviceError = {
+    error: string;
+    device?: TrezorDevice;
+    duplicate?: TrezorDevice;
+};
+type AuthorizeDeviceSuccess = { device: TrezorDevice; state: string };
+
+export const authorizeDevice = createThunk<
```

Comment:
nit: Could you please rename this with `Thunk` suffix? I know it wasn't like that before, but I think we can gradualy change those parts of the code, that are touched. But if you don't want to do it so it stay consistent with rest of the file, I get it. It is just easier to understand it in context of other filex.

```suggestion
export const authorizeDeviceThunk = createThunk<
```

Reply:
[5873524](https://github.com/trezor/trezor-suite/pull/12438/commits/5873524b456b5680a1ad46c4cdcdc579bf5f9dc2)